Jenkinsfile

```
@@ -78,7 +80,7 @@ def rpm_test_daos_test = '''me=\\\$(whoami)
                                 echo \"Server stdout: \\\$line\"
                             done
                             echo \"Server started!\"
-                            daos_agent &
+                            daos_agent -o /usr/etc/daos_gent.yml -i &
```

```suggestion
                            daos_agent -o /usr/etc/daos_agent.yml -i &
```
If I'm not misunderstanding.

---

Hmm how did this pass jenkins tests then?

---

Yes, exactly what I was asking above:

> But why didn't the RPM test stages fail with the agent starting without a valid agent file?

It's a bit concerning that a required configuration file, being missing does not prevent things from working.